Project.toml

```
@@ -1,31 +1,30 @@
 name = "ReferenceTests"
 uuid = "324d217c-45ce-50fc-942e-d289b448e8cf"
 authors = ["Christof Stocker <stocker.christof@gmail.com>", "Lyndon White <oxinabox@ucc.asn.au>"]
-version = "0.7.0"
+version = "0.7.1"
```

```suggestion
version = "0.8.0"
```

Is breaking thus needs to bump leading nonzero